pkg/ovs/openflow/ofctrl_bridge.go

```diff
@@ -502,6 +526,58 @@ func (b *OFBridge) AddOFEntriesInBundle(addEntries []OFEntry, modEntries []OFEnt
 	return nil
 }
 
+// SubscribePacketIn registers a consumer to listen PacketIn message with a provided reason. When Bridge
```

Comment:
same suggestion as above for the comment since it is copied from the interface

is it a good idea to duplicate comments between the interface and the implementation? makes it harder to maintain.